js/app.js

```diff
@@ -218,6 +218,106 @@
 			}.bind(this));
 		},
 
+		onAudioOutputChange: function() {
+			localStorage.setItem("audioOutput", $('#audioOutput').val());
+
+		},
+
+		onAudioSourceChange: function() {
+			localStorage.setItem("audioSource", $('#audioSource').val());
```

Comment:
```suggestion
			localStorage.setItem('audioSource', $('#audioSource').val());
```

Please use single quotes everywhere